webui/react/src/hooks/useTheme.ts

```diff
@@ -51,13 +52,14 @@ const updateAntDesignTheme = (path: string) => {
  * and storybook Theme decorators and not individual components.
  */
 export const useTheme = (): ThemeHook => {
+  const currentTheme = (() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
```

Comment:
**super nit:** `currentMode`? theme I think of as the actual theme data we return.